src/snippets/contact-form.njk

```diff
@@ -0,0 +1,29 @@
+---
+title: Contact Form
+contributorsName: Ryan Trimble
+contributorsURL: https://www.ryantrimble.com/
+templateEngineOverride: md, njk
+---
+{% set description %}
+Generally you will need some sort of backend process to handle form submissions. You can handle this on your own on the [server side](https://developer.mozilla.org/en-US/docs/Learn/Forms/Sending_and_retrieving_form_data) by setting the `action` and `method` attributes, or you can utilize a third-party solution such as what [Netlify](https://docs.netlify.com/forms/setup/) provides.
+
+Make sure to include labels for each input, as well as setting the proper type of input. You can also mark inputs as required by including the `required` attribute. Modern browsers have built-in (though fairly basic) form validation.
+{%- endset -%}
+{%- set html -%}
+<form>
+  <label for="full-name">
```

Comment:
This is an excellent idea! I think I would like to promote the pattern of the label as separate from the input for these field types vs. wrapping the input, mostly due to greater flexibility in styling. Would you mind making that adjustment?